website/docs/r/cse_custom_entity_type.html.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_cse_custom_entity_type"
+description: |-
+  Provides a CSE Custom Entity Type
+---
+
+# sumologic_cse_custom_entity_type
+Provides a CSE Custom Entity Type.
+
+## Example Usage
+```hcl
+resource "sumologic_cse_custom_entity_type" "custom_entity_type" {
+  name = "New Custom Entity Type"
+  identifier = "identifier"
+  fields =["file_hash_md5", "file_hash_sha1"]
+}
+```
+
+## Argument reference
+
+The following arguments are supported:
+
+- `name` - (Required) Human friend and unique name. Examples: "Ip Address", "Username", "Mac Address".
+- `identifier` - (Required) Machine friendly and unique identifier. Examples: "ip", "username", "mac".
```

Comment:
"username","ip","mac", won't actually be valid here. Let's go with the example of `name="File Hash"`, `identifier="filehash"`

Reply:
done under:
https://github.com/SumoLogic/terraform-provider-sumologic/pull/275/commits/0d141b9253b64f0aa25854a15d86d0f1ddfb1fbb